fe/fe-core/src/main/java/com/starrocks/sql/ast/SubmitTaskStmt.java

```diff
@@ -56,6 +56,9 @@ public SubmitTaskStmt(TaskName taskName, int sqlBeginIndex, DataCacheSelectState
         this.taskName = taskName.getName();
         this.sqlBeginIndex = sqlBeginIndex;
         this.dataCacheSelectStmt = dataCacheSelectStatement;
+        if (this.dataCacheSelectStmt.getProperties() != null) {
+            this.properties.putAll(this.dataCacheSelectStmt.getProperties());
```

Comment:
is there any sensitive information that needs to be masked out?

Reply:
no